src/lib/components/ControlPad.svelte

```diff
@@ -84,24 +85,36 @@
 </script>
 
 <div aria-label="button container" class="container mx-auto py-5">
-	{#each game.currentRack.gameBalls as ball}
-		{#if ball.isDead}
-			<button
-				class="relative flex items-center justify-center"
-				on:click={() => removeDeadBall(ball)}
-				disabled={!isDeadBallMode}
-			>
-				<Ball {ball} size="large" />
-				<SkullyIcon class={'w-[9rem] -m-6'} />
-			</button>
-		{:else}
-			<button
-				on:click={() => handleBallClick(ball)}
-				disabled={ball.isDead || (isGameOver && !isDeadBallMode)}
-			>
-				<Ball {ball} size={'large'} />
-			</button>
-		{/if}
+	{#each game.currentRack.gameBalls as ball (ball.number)}
+		<div class="relative">
+			{#if ball.isDead}
+				<button
```

Comment:
<button
					class="relative"
					class:hidden={!ball.isDead}
					on:click={() => removeDeadBall(ball)}
					disabled={!isDeadBallMode}
					in:send|global={{ key: ball.number }}
					out:receive|global={{ key: ball.number }}
					on:transitionend={(e) => e.currentTarget.classList.remove('absolute')}
				>